backend/src/main/java/de/neuefische/henny/reisekasse/model/dto/AddTravelFundUserDto.java

```diff
@@ -0,0 +1,17 @@
+package de.neuefische.henny.reisekasse.model.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+public class AddTravelFundUserDto {
```

Comment:
Do you mean "AddTravelF**o**undUserDto"?

I don't see no need for this dto since it holds the same attributes as in the model